src/main/haskell/kore/src/Kore/Step/AxiomPatterns.hs

```diff
@@ -27,6 +28,7 @@ it can only represent axioms that look like left-pattern => right-pattern.
 data AxiomPattern level = AxiomPattern
```

Comment:
Update the comments accordingly; probably `left-pattern => right-pattern` => `left-pattern => right-pattern requires requires-pattern`?